src/wallet/wallet.cpp

```diff
@@ -2167,10 +2182,16 @@ bool CWallet::SelectCoins(const vector<COutput>& vAvailableCoins, const CAmount&
             ++it;
     }
 
+    size_t nLimitAncestors = std::min(GetArg("-limitancestorcount", DEFAULT_ANCESTOR_LIMIT), GetArg("-limitdescendantcount", DEFAULT_DESCENDANT_LIMIT));
+    bool frejectLongChains = GetBoolArg("-rejectlongwalletchains", DEFAULT_REJECT_TOOLONG_CHAINS);
```

Comment:
Might be best to have the default const, variable, and argument be more closely matched.

Suggest `DEFAULT_WALLET_REJECT_LONG_CHAINS`, `fRejectLongChains`, and `-walletrejectlongchains`